### DISCUSSION TITLE
FR: Return Arrow Always Visible

### DISCUSSION THREAD

**Discussion by SHogenboom** [gsantner/markor#1933] [Ideas] [answered]
I wanted to request the possibility to have the ‘return arrow’ always visible in the menu. It shows when you’re adjusting Settings, but disappears again in all other views (Notebook; editor view). Took me quite a while to figure out that I had to use my device to go back rather than a button in the App. 

As the behavior already exists for settings, I would expect it to be a small adjustment to have it in other views as well. I tried looking at the code to see if I could find & copy the code myself. Unfortunately, without Java experience and lack of comments in the code I could not find the necessary snippet. Hoping that someone has the time to implement this feature - really think it would be a quick but super nice improvement!

**Comment by harshad1** [comment, marked as answer]
It used to be there. We (I) removed it to create more space for other things - the notebook titlebar, for example, has:

1. The note name
2. Undo and redo buttons
3. The view mode button
4. The save / save state button
5. The search button
6. The overflow menu button

It's still in the settings activity because it was never removed from there.

IMO we don't have space for it and don't need it as it duplicates core device functionality (i.e. the back button, which has been a primary navigation paradigm in android since the very beginning)
